pandas/core/config_init.py

```diff
@@ -214,7 +214,8 @@
 
 def mpl_style_cb(key):
     import sys
-    from pandas.tools.plotting import mpl_stylesheet
+    from pandas.tools.plotting import mpl_stylesheet, _update_stylesheet
+    _update_stylesheet()
```

Comment:
This is pretty hacky, but `mpl_stylesheet` is a dict defined in pandas.tools.plotting. I have my matplotlib version check in `pandas.util.testing`, but I can't run code from that in the main body of `pandas.tools.plotting` (I'm assuming a circular import).